ansible_collections/arista/avd/roles/eos_designs/python_modules/base/avdstructuredconfig.py

```diff
@@ -59,6 +59,9 @@ def router_bgp(self) -> dict | None:
         )
         platform_bgp_update_wait_install = get(self.shared_utils.platform_settings, "feature_support.bgp_update_wait_install", default=True) is True
 
+        default_maximum_paths = 16 if self.shared_utils.wan_role is not None else 4
+        default_ecmp = None if self.shared_utils.wan_role is not None else 4
```

Comment:
change for readability to 

```
if wan role
  16/None
else:
  4/4

```

Reply:
add documentation somewhere (bgp_ecmp / bgp_maximum_paths keys + preview document)